service/history/configs/quotas.go

```diff
@@ -36,81 +36,14 @@ const (
-		"/temporal.server.api.historyservice.v1.HistoryService/RespondWorkflowTaskCompleted":           1,
-		"/temporal.server.api.historyservice.v1.HistoryService/RespondWorkflowTaskFailed":              1,
-		"/temporal.server.api.historyservice.v1.HistoryService/ScheduleWorkflowTask":                   1,
-		"/temporal.server.api.historyservice.v1.HistoryService/VerifyFirstWorkflowTaskScheduled":       1,
-		"/temporal.server.api.historyservice.v1.HistoryService/SignalWithStartWorkflowExecution":       1,
-		"/temporal.server.api.historyservice.v1.HistoryService/SignalWorkflowExecution":                1,
-		"/temporal.server.api.historyservice.v1.HistoryService/StartWorkflowExecution":                 1,
-		"/temporal.server.api.historyservice.v1.HistoryService/SyncActivity":                           1,
-		"/temporal.server.api.historyservice.v1.HistoryService/SyncShardStatus":                        1,
-		"/temporal.server.api.historyservice.v1.HistoryService/TerminateWorkflowExecution":             1,
-		"/temporal.server.api.historyservice.v1.HistoryService/GenerateLastHistoryReplicationTasks":    1,
-		"/temporal.server.api.historyservice.v1.HistoryService/GetReplicationStatus":                   1,
-		"/temporal.server.api.historyservice.v1.HistoryService/DeleteWorkflowVisibilityRecord":         1,
-		"/temporal.server.api.historyservice.v1.HistoryService/UpdateWorkflowExecution":                1,
-		"/temporal.server.api.historyservice.v1.HistoryService/PollWorkflowExecutionUpdate":            1,
-		"/temporal.server.api.historyservice.v1.HistoryService/ExecuteMultiOperation":                  1,
-		"/temporal.server.api.historyservice.v1.HistoryService/StreamWorkflowReplicationMessages":      1,
-		"/temporal.server.api.historyservice.v1.HistoryService/SyncWorkflowState":                      1,
-		"/temporal.server.api.historyservice.v1.HistoryService/GetWorkflowExecutionHistory":            1,
-		"/temporal.server.api.historyservice.v1.HistoryService/GetWorkflowExecutionHistoryReverse":     1,
-		"/temporal.server.api.historyservice.v1.HistoryService/GetWorkflowExecutionRawHistory":         1,
-		"/temporal.server.api.historyservice.v1.HistoryService/GetWorkflowExecutionRawHistoryV2":       1,
-		"/temporal.server.api.historyservice.v1.HistoryService/ForceDeleteWorkflowExecution":           1,
-		"/temporal.server.api.historyservice.v1.HistoryService/GetDLQTasks":                            1,
-		"/temporal.server.api.historyservice.v1.HistoryService/DeleteDLQTasks":                         1,
-		"/temporal.server.api.historyservice.v1.HistoryService/AddTasks":                               1,
-		"/temporal.server.api.historyservice.v1.HistoryService/ListQueues":                             1,
-		"/temporal.server.api.historyservice.v1.HistoryService/ListTasks":                              1,
-		"/temporal.server.api.historyservice.v1.HistoryService/CompleteNexusOperation":                 1,
```

Comment:
hmmm, looks like caller type is not specified for CompleteNexusOperation

Reply:
The logic will default the priority to 1, so actually increases the priority for this api, as other background task processing requests will be using 2. But this should be fixed.